config/jobconfig.toml

```diff
@@ -55,8 +55,8 @@
 [CARGO_TECHNICIAN]
 "# Playtime Requirements" = 0
 "# Required Account Age" = 0
-"# Spawn Positions" = 2
-"# Total Positions" = 3
+"# Spawn Positions" = 3
+"# Total Positions" = 5
```

Comment:
it's preferred to use the "Generate Job Config" verb in the "Server" tab to generate these TOML files (unless you did do that and it's impossible to tell from the diff), just so you know for the future or if someone adds a new job and needs help adding it to the config.